linux/keyman-config/keyman_config/ibus_util.py

```diff
@@ -10,11 +11,12 @@
 
 def get_ibus_bus():
     try:
-        for i in range(10000):
+        for i in range(5):
             bus = IBus.Bus()
             if bus.is_connected() and bus.is_global_engine_enabled():
                 return bus
             bus.destroy()
+            time.sleep(1)
```

Comment:
So this may take up to 5 seconds?

Reply:
Yes. I guess this could happen during startup, so waiting 1s between tries seemed reasonable.